admin_guide/out_of_resource_handling.adoc

```diff
@@ -12,39 +12,120 @@ toc::[]
+
+A node must maintain stability when available compute resources are low.
 This is especially important when dealing with incompressible resources such as
 memory or disk. If either resource is exhausted, the node becomes unstable.
 
+Administrators can proactively monitor nodes for and prevent against situations where the node runs out of compute and memory resources using configurable xref:out-of-resource-eviction-policy[eviction policies]. 
+
+This topic also provides information on how {product-title} handles out-of-resource conditions and provides an xref:out-of-resource-schedulable-resources-and-eviction-policies[example scenario] and xref:out-of-resource-recommended-practices[best practices]:
+
+* xref:out-of-resource-reclaiming-node-level-resources[Resource reclaiming]
+* xref:out-of-resource-eviction-of-pods[Pod eviction]
+* xref:out-of-resource-scheduler[Pod scheduling]
+* xref:out-of-resource-node-out-of-resource-and-out-of-memory[Out of Resource and Out of Memory Killer]
+
 [WARNING]
 ====
-Failure to disable swap memory makes the node not recognize it is under
+If swap memory is enabled for a node, that node will not be able to detect that it is under
 *MemoryPressure*.
 
 To take advantage of memory based evictions, operators must
 xref:../admin_guide/overcommit.adoc#disabling-swap-memory[disable swap].
 ====
 
+
 [[out-of-resource-eviction-policy]]
-== Eviction Policy
+== Configuring Eviction Policies
+
+An _eviction policy_ allows a node to fail one or more pods when the node is running low on available resources. 
+Failing a pod allows the node to reclaim needed resources.  
+
+An eviciton policy is a combination of an xref:out-of-resource-eviction-signals[eviction trigger signal] with a specific xref:out-of-resource-eviction-thresholds[eviction threshold value], that is set
+in the xref:out-of-resource-eviction-policy[node configuration file] or through the xref:out-of-resource-eviction-policy-cli[node command line]. Evictions can be either hard, where a node takes immediate action on a pod that exceeds a threshold, or soft, where a node allows a grace period before taking action. Review the sections below for important information the differences between xref:out-of-resource-eviction-thresholds[hard and soft evictions].
+
+By using well-configured eviction policies, a node can proactively monitor for and prevent
+against total starvation of a compute resource. 
+
+[NOTE]
+====
+When the node fails a pod, it terminates all containers in the pod, and
+the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[`PodPhase`] is transitioned to *Failed*.
+====
+
+[[out-of-resource-create-config]]
+=== Using the Node Configuration to Create a Policy 
+
+To configure an eviction policy using the node configuration file:
 
-Using eviction policies, a node can proactively monitor for and prevent
-against total starvation of a compute resource.
+. Edit the node configuration file on a node.
```

Comment:
I think the above step can be merged with the step below. Something like:

"Edit the edit the eviction thresholds under the `eviction-hard` or `eviction-soft` parameters in a node configuration file:"